test/libsolidity/semanticTests/abiEncoderV1/abi_decode_static_array.sol

```diff
@@ -12,3 +12,7 @@ contract C {
 // compileViaYul: also
 // ----
 // f(bytes): 0x20, 0xc0, 0x1, 0x2, 0x3, 0x4, 0x5, 0x6 -> 1, 2, 3, 4, 5, 6
+// gas Legacy: 30262
```

Comment:
Doesn't this depend on the evm version?

Reply:
We decided to go with latest EVM, Will change gas costs to match latest